src/arrays/ArrayUtil.java

```diff
@@ -46,49 +63,83 @@ public static boolean contains(int[] array, int value) {
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                return i;
+            }
+        }
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
-
+        int newArray[] = new int[array.length - 1];
+        for (int i = 0; i < index; i++) {
+            newArray[i] = array[i];
+        }
+
+        for (int i = index; i <array.length - 1; i++) {
+            newArray[i] = array[i + 1];
+        }
+
+        for (int i = 0; i < newArray.length; i++) {
+            System.out.println(newArray[i]);
+        }
     }
```

Comment:
It's ok too, but you can write this method use one for, and if. for example`  

int result[] = new int[array.length - 1];
        for (int i = 0, j = 0; i < array.length; i++) {
            if (i != index) {
                result[j++] = array[i];
            }
        }